community-packages/generate_package_list.py

```diff
@@ -1,3 +1,5 @@
+#! /usr/bin/env python3
+
 import json
```

Comment:
I think I have dupe imports here - can you clean them up?

Reply:
sure thing!